src/app.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

```

Comment:
Change this to:
``
import { MuiThemeProvider, getMuiTheme, colors } from 'material-ui';
``